README.md

```diff
@@ -24,6 +29,168 @@ From the root of the cloned project, enter the following in your command line to
 npm ci
 ```
 
+## Supported Netlify Endpoints
```

Comment:
I think it would make more sense to put this section at the end of the readme. The front of the readme should prioritise in order the information needed to start working with the repo from scratch. It might even be better to put this info in a separate docs file inside the "functions" directory itself if this is possible without confusing the deploy.